### PR TITLE
Rdm 2420 - Delete User profiles

### PR DESCRIPTION
**Pull request by mohammedanas**
https://tools.hmcts.net/jira/browse/RDM-2420

Users will have access to more than one jurisdiction (RDM-2552) and should not be deleted from the database entirely, instead they will be deleted from the mapping between user and jurisdiction.
This will ensure users that have access to other jurisdictions do not loose that access.

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/public/stylesheets/app.scss [hunk 1]
```diff
@@ -1,168 +1,203 @@
...
+  overflow: hidden;
+  white-space: normal;
+ }
+
+.disableborder {
+  border-color: darkgray;
+}
+
+  .govuk-fieldset__legend--xl {
+    font-family: "nta", Arial, sans-serif;
```
Comment (adr1ancho) [inline src/main/public/stylesheets/app.scss]: I am not sure about the indentation here.  Are we having one too many

File: src/main/public/stylesheets/app.scss [hunk 2]
```diff
@@ -1,168 +1,203 @@
...
+$nta-font-path: '/node_modules/@govuk/platform-template/assets/fonts';
+
+
+// PDK's footer fix
+#footer .footer-meta {
+  .footer-meta-inner .open-government-licence .logo a {
+    background: url('#{$govuk-template-image-path}/open-government-licence.png') no-repeat;
+  }
+
+}
```
Comment (adr1ancho) [inline src/main/public/stylesheets/app.scss]: Do we need this blank line

File: src/test/service/delete.user.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      get: sinon.stub(),
+    };
+    config.get.withArgs("adminWeb.userprofiles_url").returns(deleteUserProfileURL);
+
+    deleteUserProfile = proxyquire("../../main/service/delete.user.profile.service", {
+      config,
+    }).deleteUserProfile;
+  });
+
+  it("should return an HTTP 201 status and success message", (done) => {
```
Comment (Daniel-Lam) [inline src/test/service/delete.user.service.spec.ts]: This isn't correct - HTTP `201` is used to indicate something has been created, but this is a deletion! It should be checking for a `204`

File: src/test/service/delete.user.service.spec.ts [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    const expectedResult = "User profile deleted successfully";
+
+    nock("http://localhost:4453")
+      .delete("/users")
+      .query({ uid: "aaa@yahii.com", jid: "test" })
+      .reply(201, expectedResult);
```
Comment (Daniel-Lam) [inline src/test/service/delete.user.service.spec.ts]: As above, this should be a `204`

File: src/test/routes/delete.user.profile.spec.ts [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+        it("should return Confirm delete user profile page when authenticated", () => {
+            idamServiceMock.resolveRetrieveUserFor("1", "admin");
+            idamServiceMock.resolveRetrieveServiceToken();
+
+            return request(app)
+                .get("/deleteuserprofile")
+                .query({ idamId: "anas@yahoo.com" })
+                .set("Cookie", "accessToken=ey123.ey456")
+                .then((res) => {
+                    expect(res.statusCode).to.equal(200);
```
Comment (Daniel-Lam) [inline src/test/routes/delete.user.profile.spec.ts]: The correct HTTP response code should be a `204` when there is no content to return

**Pull request merged**